package.json

```diff
@@ -118,7 +119,7 @@
     ]
   },
   "volta": {
-    "node": "8.17.0",
-    "yarn": "1.19.1"
+    "node": "10.19.0",
+    "yarn": "1.22.0"
```

Comment:
cc @bendemboski do you need us to cut a new major version, or is a minor version bump sufficient for you?

Reply:
I guess we need a major -- we depend on `^3.12.5` and haven't dropped node 8 support yet (and our 2.x/non-beta version also depends on `^3.12.5`), so a minor bump dropping node 8 support would break our node 8 support.

Reply:
Thanks for checking!